packages/tools/client-debugger/src/Registry.ts

```diff
@@ -0,0 +1,252 @@
+/*!
```

Comment:
This file is showing as a new addition in the diff, but it should already exist. Have you merged main / rebased onto it?

Reply:
Yea. It sync-ed from latest main branch.

Reply:
The file still appears as being added here. You might need to manually delete it.